libs/language-server/src/stdlib/builtin-constrainttypes/LengthConstraint.jv

```diff
@@ -0,0 +1,17 @@
+// SPDX-FileCopyrightText: 2023 Friedrich-Alexander-Universitat Erlangen-Nurnberg
+//
+// SPDX-License-Identifier: AGPL-3.0-only
+
+/**
+* Limits the length of a string with an upper and/or lower boundary. Only values with a length within the given range are valid.
+*
+* @example A string with 0 to 2147483647 characters.
```

Comment:
Including 0 and the max? Or not including?

Reply:
Also, why this weird number in an example. Why not 1-10 for the VIN10 number we have in another jayvee example?